pages/codeshare/post/[urlTitle].tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
```

Comment:
We need to remove the `post` section here.

We need the URL to be the same as now.